proto/server/configEvtHandler.go

```diff
@@ -196,6 +185,22 @@ func handleDeviceGroupPost(configMsg *configmodels.ConfigMessage, subsUpdateChan
 	rwLock.Unlock()
 }
 
+func handleDeviceGroupDelete(configMsg *configmodels.ConfigMessage, subsUpdateChan chan *Update5GSubscriberMsg) {
+	rwLock.Lock()
+	if factory.WebUIConfig.Configuration.Mode5G {
+		var config5gMsg Update5GSubscriberMsg
+		config5gMsg.Msg = configMsg
+		config5gMsg.PrevDevGroup = getDeviceGroupByName(configMsg.DevGroupName)
+		subsUpdateChan <- &config5gMsg
```

Comment:
I wonder was this code missing from old code? We were not writing event on channel?

Reply:
It was always there. I just extracted the code responsible for deleting network slices and device groups to separate methods. IMO it reads better and makes it consistent with the rest of the code.

Reply:
I have the same question as Ajay, if the code was simply refactored, there should be an equivalent line removed elsewhere no?

Reply:
Line 141-144 in your branch are writing to the channel, so with your change as it is right now, each change would be sent to the channel twice.

Reply:
Ahh, of course. Copy-pasting backfired on me. 
Sorry for the confusion. I removed the entire `if` from both new functions, so that writing to the channel is handled centrally for all deletions.